numpy/lib/function_base.py

```diff
@@ -660,11 +660,11 @@ def select(condlist, choicelist, default=0):
     >>> x = np.arange(6)
     >>> condlist = [x<3, x>3]
     >>> choicelist = [x, x**2]
-    >>> np.select(condlist, choicelist, 55)
-    array([ 0,  1,  2, 55, 16, 25])
+    >>> np.select(condlist, choicelist, 42)
```

Comment:
`42` is a better default value than `55`, simply because it is the answer to life, the universe, and everything :yum: